code/game/objects/structures/stool_bed_chair_nest/bed.dm

```diff
@@ -334,16 +333,6 @@ GLOBAL_LIST_EMPTY(activated_medevac_stretchers)
 
 /obj/structure/bed/medevac_stretcher/Destroy()
 	QDEL_NULL(radio)
-	if(stretcher_activated)
-		stretcher_activated = FALSE
-		GLOB.activated_medevac_stretchers -= src
-		if(linked_medevac)
-			linked_medevac.linked_stretcher = null
-			linked_medevac = null
-		update_icon()
-	if(linked_beacon)
-		linked_beacon.linked_bed_deployed = null
-		linked_beacon = null
```

Comment:
why are you deleting beacon code here?

Reply:
Got caught with the rest of the linked bed removals, amended.